components/VM_Arm/src/main.c

```diff
@@ -1156,6 +1157,9 @@ int main_continued(void)
     ZF_LOGF_IF(err, "Failed to bind CB to SID");
 #endif /* CONFIG_ARM_SMMU */
 
+    vm.fdt_ori = (void *)ps_io_fdt_get(&_io_ops.io_fdt);
+    vm.gic_path = GIC_NODE_PATH;
```

Comment:
Thanks for splitting the PRs up. Please add the usage of `vm->gic_path` in `generate_fdt()` also, so this is the only place where the path's default is set up from the plat constants.